autogen/safer-cluster/main.tf.tmpl

```diff
@@ -119,7 +119,7 @@ module "gke" {
   //   All applications should run with an identity defined via Workload Identity anyway.
   // - Use a service account passed as a parameter to the module, in case the user
   //   wants to maintain control of their service accounts.
-  create_service_account = var.compute_engine_service_account == "" ? true : false
+  create_service_account = var.create_service_account == false ? var.create_service_account : (var.compute_engine_service_account == "" ? true : false)
```

Comment:
Allowing `create_service_account` to be set explicitly rather than depending on the value of `var.compute_engine_service_account` fixes the error.